helix-term/src/commands/typed.rs

```diff
@@ -910,9 +910,30 @@ pub(super) fn goto_line_number(
     Ok(())
 }
 
+// Fetch the current value of a config option and output as status.
+fn get_option(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    _event: PromptEvent,
+) -> anyhow::Result<()> {
+    if args.len() != 1 {
+        anyhow::bail!("Bad arguments. Usage: `:get key`");
+    }
+
+    let key = &args[0].to_lowercase();
+    let key_error = || anyhow::anyhow!("Unknown key `{}`", key);
+
+    let mut config = serde_json::to_value(&cx.editor.config().clone()).unwrap();
+    let pointer = format!("/{}", key.replace('.', "/"));
+    let value = config.pointer_mut(&pointer).ok_or_else(key_error)?;
```

Comment:
`pointer()` should work here instead of `pointer_mut()` since we're only reading.

Reply:
My mistake. Good catch @sudormrfbin. Thanks :)